pkg/cri/server/instrumented_service.go

```diff
@@ -878,7 +878,11 @@ func (in *instrumentedService) UpdateContainerResources(ctx context.Context, r *
 	if err := in.checkInitialized(); err != nil {
 		return nil, err
 	}
-	log.G(ctx).Infof("UpdateContainerResources for %q with %+v", r.GetContainerId(), r.GetLinux())
+	var update interface{} = r.GetLinux()
+	if update == nil {
+		update = r.GetWindows()
+	}
```

Comment:
Is this just used for type checking? Do we need to do this here?

Reply:
It's just for logging below. Previously, it was only logging r.GetLinux(), which was not helping on Windows. This will have to be updated for LCOW.

Reply:
That's fine, I was just curious